scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/BaseOxfordNanoporeProcessor.java

```diff
@@ -263,11 +260,12 @@ public synchronized NotificationDto process(File runDirectory, TimeZone tz) thro
         String sequencingKit = genericReader.string().getAttr(contextTags, "sequencing_kit");
         if (!(sequencingKit.equals(""))) onnd.setSequencingKit(sequencingKit);
       }
-
-      final File summaryFile = new File(runDirectory, "final_summary.txt");
-      if (summaryFile.exists()) {
+      final Optional<File> summaryFile = Stream.of(Objects.requireNonNull(runDirectory.listFiles()))
+              .filter(f -> f.getName().matches("final_summary.*\\.txt"))
+              .findFirst();
+      if (summaryFile.isPresent() && summaryFile.get().exists()) {
```

Comment:
The `summaryFile.get().exists()` is superfluous now since the `listFiles` will only show files that exist.

Reply:
Thanks for pointing that out! Removed in 8e34d43